### PR TITLE
planner: update the fix-control doc

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by winoros — COMMENTED** [review]
the fixcontrol#46177 also needs a update

**Review by winoros — APPROVED** [review]

File: optimizer-fix-controls.md [hunk 1]
```diff
@@ -71,6 +71,18 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 - This variable sets the threshold for the optimizer's heuristic strategy to select access paths. If the estimated rows for an access path (such as `Index_A`) is much smaller than that of other access paths (default `1000` times), the optimizer skips the cost comparison and directly selects `Index_A`.
 - `0` means to disable this heuristic strategy.
 
+### [`45798`](https://github.com/pingcap/tidb/issues/45798) <span class="version-mark">New in v7.5.0</span>
+
+- Default value: `ON`
+- Possible values: `ON`, `OFF`
+- This variable controls whether to allow the Plan Cache to cache plans accessing generated columns.
+
+### [`46177`](https://github.com/pingcap/tidb/issues/46177) <span class="version-mark">New in v7.5.0</span>
```
Comment (Oreoxmt) [inline optimizer-fix-controls.md:80]:
```suggestion
### [`46177`](https://github.com/pingcap/tidb/issues/46177) <span class="version-mark">New in v6.5.6, v7.1.3 and v7.5.0</span>
```

File: optimizer-fix-controls.md [hunk 2]
```diff
@@ -71,6 +71,18 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 - This variable sets the threshold for the optimizer's heuristic strategy to select access paths. If the estimated rows for an access path (such as `Index_A`) is much smaller than that of other access paths (default `1000` times), the optimizer skips the cost comparison and directly selects `Index_A`.
 - `0` means to disable this heuristic strategy.
 
+### [`45798`](https://github.com/pingcap/tidb/issues/45798) <span class="version-mark">New in v7.5.0</span>
+
+- Default value: `ON`
+- Possible values: `ON`, `OFF`
+- This variable controls whether to allow the Plan Cache to cache plans accessing generated columns.
```
Comment (Oreoxmt) [inline optimizer-fix-controls.md:78]:
```suggestion
- This variable controls whether Plan Cache is allowed to cache execution plans that access [generated columns](/generated-columns.md).
```

File: optimizer-fix-controls.md [hunk 3]
```diff
@@ -71,6 +71,18 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
...
+
+### [`46177`](https://github.com/pingcap/tidb/issues/46177) <span class="version-mark">New in v7.5.0</span>
+
+- Default value: `ON`
+- Possible values: `ON`, `OFF`
+- This variable controls whether to explore enforced plans during optimization if it has already found an unenforced plan.
```
Comment (Oreoxmt) [inline optimizer-fix-controls.md:84]:
```suggestion
- This variable controls whether the optimizer explores enforced plans during query optimization after finding an unenforced plan.
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**